### ISSUE TITLE
2022-09-28 Patch (security) releases plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1945]
## 2.5.0-beta2
- #1937 
- #1944 

## 2.4.4
- #1937 
- #1944 

## 2.3.11
- #1943

**Comment by joeparsons** [comment]
# Pick lists

## 2.5.x
```
git cherry-pick 56eb2367b4bb0c5f28ed7f6814b724b50d0e5144
git cherry-pick f58a3f70279349cc660b08e093f1d267f781d280
```

## 2.4.x
```
git cherry-pick 56eb2367b4bb0c5f28ed7f6814b724b50d0e5144
git cherry-pick f58a3f70279349cc660b08e093f1d267f781d280
```

## 2.3.x
(None needed)

**Comment by joeparsons** [comment]
# Release note snippets

## 2.5.0-beta2

`git log 2.5.0-beta1..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for Drupal core addressing a critical security advisory for multiple vulnerabilities ([SA-CORE-2022-016](https://www.drupal.org/sa-core-2022-016))._

_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2. Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

- #1942: Update Drupal core to 9.4.7 for SA-CORE-2022-016 (critical security update). (#1944)
- Closes #1936: Add patch to fix CAS module session handling on Drupal 9.2+. (#1937)
- Back to dev.
```

## 2.4.4

`git log 2.4.3..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for Drupal core addressing a critical security advisory for multiple vulnerabilities ([SA-CORE-2022-016](https://www.drupal.org/sa-core-2022-016))._

- #1942: Update Drupal core to 9.4.7 for SA-CORE-2022-016 (critical security update). (#1944)
- Closes #1936: Add patch to fix CAS module session handling on Drupal 9.2+. (#1937)
- Update composer branch alias in preparation for 2.5.0 pre-release.
- Back to dev.
```


## 2.3.11

`git log 2.3.10..HEAD --pretty='format:- %s'`
```
_Note: This release contains a security update for Drupal core addressing a critical security advisory for multiple vulnerabilities ([SA-CORE-2022-016](https://www.drupal.org/sa-core-2022-016))._

- #1942: Update Drupal core to 9.3.22 for SA-CORE-2022-016 (critical security update). (#1943)
- Update composer branch alias in preparation for 2.5.0 pre-release.
- Closes #1903: Tag correct revision during 'Create release' GH actions workflow. (#1905)
- Back to dev.
```

**Comment by joeparsons** [comment]
Releases created:
https://github.com/az-digital/az_quickstart/releases
